e2e-test/features/DesignTimeSLT.feature

```diff
@@ -0,0 +1,65 @@
+# Copyright © 2021 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+Feature: Design Time SLT Scenario
+
+  @SLT @Design-Time @Sanity
+  Scenario Outline: User is able to provides input to SLT parameters
+    Given Open CDF replication and initiate pipeline creation
+    When Source is SAP SLT
+    Then User is able to set SLT parameter "<option>" as "<input>"
+    Examples:
+      | option            | input         |
+      | gcsDataPath       | invalid       |
+      | guid              | invalidguid   |
+      | massTransferId    | ZZZ66         |
+      | sapJcoLibGcsPath  |gs://ga_slt_cdf|
+      | jco.client.ashost | invalid       |
+      | jco.client.sysnr  | abc           |
+      | jco.client.client | abc           |
+      | jco.client.lang   | invalid       |
+      | jco.client.user   | invalid       |
+      | jco.client.passwd | invalid       |
+
+  @SLT @Design-Time @Sanity
+  Scenario: User is able to set Replicate Existing Data as false
+    Given Open CDF replication and initiate pipeline creation
+    When Source is SAP SLT
+    Then Replicate Existing Data is set to false
```

Comment:
-> Then verify Replicate Existing Data is set to false

Reply:
Working in progress.